app/CocoaPods.xcodeproj/project.pbxproj

```diff
@@ -591,9 +632,20 @@
 			baseConfigurationReference = CBE7AF19067EFC56753C3D79 /* Pods.debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CODE_SIGN_IDENTITY = "";
```

Comment:
it might be, because in https://github.com/CocoaPods/CocoaPods-app/pull/81 we made it so that this happens at the Makefile level